packages/hub/indexers.js

```diff
@@ -182,7 +182,7 @@ class Indexers {
     } finally {
       running.destroy();
     }
-    this.events.emit('index_update');
+    this.events.emit('index_update', hints);
```

Comment:
@ef4 I added the hints to this event for now. from the indexer it doesnt look like I have direct access to the operations. The event listeners need to get access to the event emitter in order to wire up correctly. Right now that happens pretty easily as there is just one `hub:indexers` that can be injected in the live-query service.